pom.xml

```
@@ -13,12 +13,12 @@
     <version>${revision}-${changelist}</version>
     <packaging>hpi</packaging>
     <properties>
-        <revision>6.8.1</revision>
+        <revision>6.9.0</revision>
```

Most recent release of upstream library.  Supersedes pull request
* #238

```suggestion
        <revision>6.9.2</revision>
```